Modules/Tokens/test/Tokens.API.Tests.Integration/Models/HttpResponse.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Net;
+
+namespace Tokens.API.Tests.Integration.Models;
+public class HttpResponse<T>
+{
+    public T? Data { get; set; }
```

Comment:
If we want to access a Token's ```Content``` property, then it would look like the following: ```httpResponse.Content.Result.Content```. By using ```Data``` it looks like the following: ```httpResponse.Data.Result.Content``` and I thought it looked better. Toughts?

Reply:
Looks weird, I agree. But I think it's still better than living with the inconsistency.

Reply:
I don't think that we need the raw Content property, do we?

Reply:
We do to validate the json schema

Reply:
Well, actually not. In the validate method, you always parse the raw string before you do the JSON schema validation. So you could as well just pass in the parsed object (aka the current Data property)

Reply:
Removed the raw Content property and renamed the generic type Data property to Content

Reply:
Looks good